src/Deposit.js

```diff
@@ -37,7 +37,7 @@ const VendingMachineContract = TruffleContract(VendingMachineJSON)
 const ECDSAKeepContract = TruffleContract(ECDSAKeepJSON)
 
 // TODO Need this configured via TBTC.
-const electrumConfig = JSON.parse(fs.readFileSync("./src/electrum-config.json"))
+const electrumConfig = require(__dirname + "/../src/electrum-config.json")
```

Comment:
As a side note, this in turn doesn't work in node proper because `require` isn't allowed from ES6 modules, at least on older versions of node.

Delightful!